conmon-rs/server/src/container_log.rs

```diff
@@ -13,36 +13,43 @@ pub struct ContainerLog {
     drivers: Vec<LogDriver>,
 }
 
-#[derive(Debug)]
 enum LogDriver {
     ContainerRuntimeInterface(CriLogger),
+    Json(JsonLogger),
+}
+
+impl std::fmt::Debug for LogDriver {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Self::ContainerRuntimeInterface(arg0) => f
+                .debug_tuple("ContainerRuntimeInterface")
+                .field(arg0)
+                .finish(),
+            Self::Json(arg0) => f.debug_tuple("Json").field(arg0).finish(),
+        }
+    }
 }
 
 impl ContainerLog {
-    /// Create a new default SharedContainerLog.
```

Comment:
Let's keep this comment.

Reply:
```
/// Create a new default SharedContainerLog.
```
can it be put back to this so it's registered as a doc for the func